src/aap_eda/settings/default.py

```diff
@@ -21,17 +23,20 @@
 
 from .post_load import post_loading
 
-EDA_SETTINGS_FILE = "/etc/eda/settings.yaml"
+EDA_SETTINGS_FILE = os.environ.get(
+    "EDA_SETTINGS_FILE", "/etc/eda/settings.yaml"
+)
+
+os.environ["EDA_MODE"] = os.environ.get("EDA_MODE", "production")
```

Comment:
if production is the default, we should be override the value in the docker-compose development envs?

Reply:
yes. docker-compose already overwrite the value to development mode